pkg/security/podsecuritypolicy/seccomp/strategy_test.go

```diff
@@ -127,26 +133,26 @@ func TestGenerate(t *testing.T) {
 		"seccomp with no default, pod annotations": {
 			pspAnnotations: allowAnyNoDefault,
 			podAnnotations: map[string]string{
-				api.SeccompPodAnnotationKey: "foo",
+				api.SeccompPodAnnotationKey: foo,
 			},
-			expectedProfile: "foo",
+			expectedProfile: foo,
 		},
 		"seccomp with default, no pod annotations": {
 			pspAnnotations:  allowAnyDefault,
 			podAnnotations:  nil,
-			expectedProfile: "foo",
+			expectedProfile: foo,
 		},
 		"seccomp with default, pod annotations": {
 			pspAnnotations: allowAnyDefault,
 			podAnnotations: map[string]string{
-				api.SeccompPodAnnotationKey: "bar",
+				api.SeccompPodAnnotationKey: bar,
 			},
-			expectedProfile: "bar",
+			expectedProfile: bar,
 		},
 	}
 	for k, v := range tests {
 		s := NewStrategy(v.pspAnnotations)
-		actual, err := s.Generate(v.podAnnotations, nil)
+		actual, err := s.Generate(v.podAnnotations, &api.Pod{})
```

Comment:
add a `*SeccompProfile` field to the test case and pass that in here, then define the test cases that check what generate returns when the field is set and the annotation isn't:
```suggestion
		actual, err := s.Generate(v.podAnnotations, &api.Pod{
			Spec: api.PodSpec{
				SecurityContext: &api.PodSecurityContext{
					SeccompProfile: v.seccompProfile,
				},
			},
		})
```

Reply:
Done :+1: